* #5: Added user guide. (#16)
exasol/virtual-schema-installer: doc/user_guide/user_guide.md

> # Virtual Schema Installer User Guide

Author: Done

> # Virtual Schema Installer User Guide

Reviewer: Did you push this change? I don't see it.

### Examples

* Installing PostgreSQL Virtual Schema with all default values:
  
  ```shell
  java -jar virtual-schema-installer-0.1.0.jar --dialect POSTGRESQL --property CATALOG_NAME='test' --property SCHEMA_NAME='postgres_schema'
  ```

* Installing MySQL Virtual Schema with customized arguments:

  ```shell
  java -jar virtual-schema-installer-0.1.0.jar --dialect mysql --jdbc-driver-name mysql-connector.jar --jdbc-driver-path target/mysql-driver --exa-host localhost --exa-port 32769 --exa-bucketfs-port 32770 --exa-schema-name MY_SCHEMA --exa-adapter-name MY_ADAPTER_SCRIPT --exa-connection-name MY_JDBC_CONNECTION --exa-virtual-schema-name MYSQL_VIRTUAL_SCHEMA_1 --source-host 172.17.0.1 --source-port 32783 --property CATALOG_NAME='MYSQL_SCHEMA' --property TABLE_FILTER='SIMPLE_TABLE' --credentials-file /tmp/installer_credentials
  ```
  
## Things to Be Aware of

* Currently, all files are uploaded to the `/buckets/bfsdefault/drivers/jdbc` bucket directory.
* The installer downloads a virtual Schema Adapter JAR from Exasol GitHub and saves it in the default temporary directory.
* The installer only creates a new Exasol schema if a schema with a specified named doesn't exist.

> * The installer creates Exasol schema, adapter, connection and virtual schema names with quotation. It means the names you specify on the console are case-sensitive.

Reviewer: ```suggestion
* The installer uses quotes when creating Exasol schema, adapter, connection and Virtual Schema name. That means the names you specify on the console are case-sensitive.
```
